src/huggingface_hub/hf_api.py

```diff
@@ -9575,6 +9579,75 @@ def list_user_following(self, username: str, token: Union[bool, str, None] = Non
         ):
             yield User(**followed_user)
 
+    def auth_check(self, repo_id: str, repo_type: Optional[str] = None, token: Union[bool, str, None] = None) -> None:
```

Comment:
```suggestion
    def auth_check(self, repo_id: str, *, repo_type: Optional[str] = None, token: Union[bool, str, None] = None) -> None:
```

(nit) adding a `*, ` forces the arguments to be keyword-only. This is useful for us for future maintenance. Adding and reordering parameters is much easier when they are keyword-only. When things are positional and keyword, adding a new one might introduce a breaking change.